src/site/includes/debug.drupal.liquid

```diff
@@ -1,4 +1,4 @@
-{% if buildtype != 'vagovprod' and omitdebug == false %}
+{% if buildtype != 'vagovprod' %}
```

Comment:
```suggestion
{% if isPreview or buildtype != 'vagovprod' and omitdebug == false %}
```

It doesn't hurt to include this in the prod preview server IMO!  Double check my boolean arithmetic in that if-statement though (not certain how Liquid will interpret this)

Reply:
True! 🙂 I think you will need parentheses, or at least my brain needs them 😂 

```liquid
{% if isPreview or (buildtype != 'vagovprod' and omitdebug == false) %}
```

Reply:
I was wondering if Liquid supports the parentheses lol